docs/src/usage.md

```diff
@@ -1,6 +1,6 @@
 # Basic usage
 
-The basic elements of the package are **intervals**, i.e. sets of real numbers (possibly including `\pm \infty`) of the form
+The basic elements of the package are **intervals**, i.e. sets of real numbers (possibly including `±` (`\pm`) and `∞` `\infty`) of the form:
```

Comment:
This was supposed to be "plus or minus infinity", i.e. equivalent to "possibly including the two extended real numbers +infinity and -infinity"